test/passing/record.ml

```diff
@@ -27,3 +27,5 @@ let _ = {(x : x) with e1; e2}
 let _ = {(x :> x) with e1; e2}
 
 let _ = {(x#x) with e1; e2}
+
+let f ?l:({f; g}) = e
```

Comment:
can you add a test with a non optional label

Reply:
done